README.md

```diff
@@ -9,6 +9,17 @@ To run this project, you have to set up a PostgreSQL database with the following
 
 This information can be found in the **application.yml** file as well.
 
+### Database modification
+#### Enums
+Two enums should be created manually, because it cannot be done automatically:
```

Comment:
Good job for adding these to the readme!
Just a side note: these can be done automatically (both creating the new types and altering the tables) using a database migration tool (eg.: liquibase).
But we don't need to introduce that :)